src/client.ts

```diff
@@ -219,18 +234,18 @@ client.on('message', (m) => void (async msg => {
 	if (lockdown) return msg.reply('The bot is restarting soon, please try again in a minute.');
 
 	const cmdID = toID(msg.content.slice(prefix.length).split(' ')[0]);
-	let command = commands.get(cmdID);
-	if (typeof command === 'string') command = commands.get(command);
+	let Command = commands.get(cmdID);
+	if (typeof Command === 'string') Command = commands.get(Command);
 	// Throw if it's another alias
-	if (typeof command === 'string') throw new Error(`Alias "${cmdID}" did not point to command.`);
-	if (!command) return;
+	if (typeof Command === 'string') throw new Error(`Alias "${cmdID}" did not point to command.`);
+	if (!Command) return;
 
 	// 100% not an alias, so it must be a command class.
-	const cmd = new (command as Constructable<BaseCommand>)(msg);
+	const cmd = new Command(msg);
 	try {
 		await cmd.execute();
 	} catch (e) {
-		await onError(e, 'A chat command crashed: ');
+		await onError((e as Error), 'A chat command crashed: ');
```

Comment:
Can you do `catch (e: Error)`, or does TS force you to cast to `unknown` or `any`?

Reply:
Forced type because of TS yeah, its annoying.
![image](https://user-images.githubusercontent.com/11671268/137533366-1a0d7caf-aabe-46b3-bc03-3ee38eee27a9.png)